tests/others/test_dependencies.py

```diff
@@ -37,3 +38,12 @@ def test_backend_registration(self):
                     elif backend == "invisible_watermark":
                         backend = "invisible-watermark"
                     assert backend in deps, f"{backend} is not in the deps table!"
+
+    def test_pipeline_imports(self):
+        import diffusers
+
+        all_classes = inspect.getmembers(diffusers, inspect.isclass)
+        for cls_name, cls_module in all_classes:
+            if "Pipeline" in cls_name:
```

Comment:
```suggestion
            if hasattr(diffusers.pipelines, cls_name):
```
might be a bit safer